data/avro-schema/src/main/java/net/corda/data/flow/CpiPropertyKeys.java

```diff
@@ -0,0 +1,10 @@
+package net.corda.data.flow;
```

Comment:
The package sounds right but the module "data/avro-schema" doesn't sound right, sounds more like we should have a module inside "data" called "flow-schema" or something. Wdyt @mnesbit ? These are properties that will be used in flows to get from `flowEngine.getContextProperties(key)`